portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,28 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Peter's Portfolio</h1>
+      <p>This is my portfolio. A page to get a breif description of me</p>
+      <img src="image/profile.PNG"/>
+      <h2>Summary</h2>
+      <p>Hello, I am peter, I am a Junior at Cal State Sacramento and I have been coding for about a year now, I am from Texas but born in Nigeria and an international student and I really enjoy building software.</p>
+      <p>I really enjoy coding particularly in java and python, I really like the idea of being able to bring an idea to real life through writing code.  love photography and I do some of that in my free time. I have included some links to my LinkedIn and GitHub profiles and some images that tell you a bit more about me</p>
+      <img src="image/beau.jpg"/>
+      <p style="text-align:center">This is Beau, he loves water for some reason.</p>
+      <img src="image/pink1.jpg"/>
+      <p style="text-align:center">A nice picture of some Japanese cherry Trees I took.</p>
```

Comment:
**No need to do this!**, but if it wasn't covered in the assignment I want to point out that you can make your own CSS classes, then tag each caption with the class. For example:

```
<div>
  <img src="image/pink1.jpg"/>
  <p class="caption"> A nice picture of some Japanese cherry Trees I took.</p>
```

Later in your CSS file

```
p.caption {
  text-align: center;
}
```

**Why do this?** In-line style (e.g. `style="text-align:center`) is great and works well. But if you find yourself applying it over and over again it's worth asking whether you think you'll want to change it in the future.

Let's say in the future you wanted to make your captions italicized. 

With in-line style you'll need to go through the whole HTML document and update every caption `<p>`  with `<p style="text-align:center;font-style:italic">`. With just a few pictures this is no big deal, but imagine if you had dozens all over the web page!

With using a special CSS class all you would need to do is update your class to be:

```
p.caption {
  text-align: center;
  font-style: italic; // 1 line change!
}
```

Reply:
Neat, that makes it a bit more fluid